### PR TITLE
Specific Exception Handling

### LINKED ISSUE #9: Properly catch and deal with "HTTPTimeoutError" exceptions
The exception handling routines currently employed in the project are too generalized. Handling routines for specific exceptions would be appreciated.

### PR DESCRIPTION
**Pull request by divyagupta-98**

### CODE REVIEW

**Review by gridhead — CHANGES_REQUESTED** [review]
Hi @divyagupta-98, as you are dealing with specifically HTTPTimeoutError - please restrict the PR changes to that only.

**Review by gridhead — CHANGES_REQUESTED** [review]
Please refrain from storing the exception in a variable.

**Comment by gridhead** [comment]
Also, could you please make the commit message for 4410050 a bit more descriptive?

**Comment by divyagupta-98** [comment]
@t0xic0der Done with the changes.

File: main.py [hunk 1]
```diff
@@ -129,7 +139,9 @@ def getimage(self):
                 for indx in range(len(purltext)):
                     click.echo(click.style("#" + str(indx+1), fg="magenta", bold=True) + " " + purltext[indx])
                 click.echo(click.style("RAISED > ", fg="green", bold=True) + str(len(purltext)) + " result(s) in " + str(duration)[0:3] + " seconds [" + self.obtntime() + "]")
-            except Exception as expt:
+            except wikipedia.exceptions.HTTPTimeoutErro:
+                click.echo(click.style("CAVEAT > ", fg="red", bold=True) + "Timeout occured while processing the query.")
```
Comment (gridhead) [inline main.py]: The exception-catching keyword here is misspelled. Please correct it.

**Review by gridhead — APPROVED** [review]
Looks good to me. Merging.

**Pull request merged**